.github/workflows/func.yaml

```diff
@@ -86,6 +86,11 @@ on:
       juju-accounts-yaml:
         required: false
         description: "A base64 encoded accounts.yaml file to use"
+      openstack-auth-env:
+        required: false
+        description: |
+          A base64 encoded openstack authentication env file.
```

Comment:
Can you add the format of the file in the description so that it's clearer for first time viewers of this option?
I imagine it would be something like this:
```
OS_AUTH_URL=http://1.1.1.1:5000/v3
OS_USERNAME=XXX
OS_PASSWORD=XXX
OS_PROJECT_NAME=XXX
......
```

Reply:
Should we?

The env file seems a common term.

Reply:
It's a nice to have. You can ignore my comment if you feel it's already intuitive.